composer.json

```
@@ -19,7 +19,8 @@
     "humanmade/authorship": "~0.2.9",
     "humanmade/altis-reusable-blocks": "~0.2.0",
     "humanmade/asset-loader": "^0.5.0",
-    "10up/simple-local-avatars": "~2.2.0"
+    "10up/simple-local-avatars": "~2.2.0",
+    "automattic/advanced-post-cache": "dev-master"
```

Only change to make here is to pin a specific commit. If we use `dev-master` then it will always fetch the latest version from the `master` branch which can change outside of our control. For stability we always want to install a known or at least trusted state of the code, this is why typically we'll use tags only, or tags with a version constraint like `^` or `~`.

```suggestion
    "automattic/advanced-post-cache": "dev-master#c46b2d95773689938ef1e89a3dfcb9cbd94022b8"
```

---

Thank you, [Resolved in this commit](https://github.com/humanmade/altis-cms/pull/468/commits/cf650b6ceba3ebbacc9c71cd997af710bd5af603)